src/lib/y2network/sysconfig/interfaces_writer.rb

```diff
@@ -105,8 +105,6 @@ def clean_up_old_interfaces(interfaces)
           set_interface_down(iface.old_name)
           ifcfg = Y2Network::Sysconfig::InterfaceFile.find(iface.old_name)
           ifcfg && ifcfg.remove
```

Comment:
is it needed? it should be done in connection config writer, not?

Reply:
Yes, now it should be covered.